### PR TITLE
make stuff gooder

### PR DESCRIPTION
**Pull request by jhford**

### CODE REVIEW

**Comment by jonasfj** [comment]
# `r+`

File: src/provision.js [hunk 1]
```diff
@@ -14,6 +14,21 @@ let series = require('./influx-series');
...
+ * the WorkerType entity, the name is input[n].workerType.workerType.  The
+ * second item is a bid.  The bids have properties region, type (InstanceType),
+ * zone and some pricing information.
+ *
+ * Do not edit the items in the list, just copy them into a new list.  Treat
+ * them as immutable.
+ */
+function orderThingsInRegion(input) {
+  return input;
+}
```
Comment (jonasfj) [inline src/provision.js]: I think this is for dustin to play with :)

Comment (jhford) [inline src/provision.js]: @imbstack also expressed an interest in that as well.

Comment (djmitche) [inline src/provision.js]: #105 :)

**Pull request merged**